edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/MetricsFileWriter.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Writes to a local file instead of uploading.
+    /// </summary>
+    public sealed class MetricsFileWriter : IMetricsPublisher
+    {
+        public async Task PublishAsync(IEnumerable<Metric> metrics, CancellationToken cancellationToken)
+        {
+            using (FileStream file = File.Open(@"/shared/MetricsUpload.txt", FileMode.Append))
```

Comment:
Is this hardcoded path okay? Does this work on Windows?

Reply:
it wouldn't. I changed it to shared/MetricsUpload.txt so it will live in app/shared/MetricsUpload.txt now. This will let it work on windows. I think hardcoded is fine since it's just intended for testing.